scripts/start_woltka

```diff
@@ -47,9 +47,12 @@ def execute(url, job_id, out_dir):
         else:
             directory = directory.pop()
 
-        # Getting all the files from the database
-        main_fp, merge_fp = woltka_to_array(
-            files, out_dir, parameters['Database'], prep, url, job_id)
+        if command.lower().startswith('woltka'):
```

Comment:
is it okay to assume woltka_syndna_to_array if command is not 'woltka' here? I'm thinking it would be a silent error if command ends up being a junk value. What do you think about checking against a set of valid strings and raising an error if something unexpected is passed?

Reply:
Sure, I can add.